tests/search-engine/test_rerank.spec.ts

```diff
@@ -55,12 +56,26 @@ const documents = [
     "항상 긍정적인 태도를 유지하려고 노력한다.",
     "오늘은 특별한 일이 있어 기분이 좋다.",
 ];
+  
+let model: any;
```

Comment:
Why are you use let to declare model and tokenizer variable here and load it in the `setupModel`?
I think it is not great pattern.

1. Don't need to be a function. This model is just test purpose right?
2. Since it is just a test, I don't know why it has to be declare as global variable in this file. Why don't we declare in the `describe` function? And directly load using `.from_pretrained` functions.